src/components/inline-link-button/index.tsx

```diff
@@ -0,0 +1,23 @@
+import { ReactElement } from 'react'
+import classNames from 'classnames'
+import { InlineLinkButtonProps } from './types'
+import s from './inline-link-button.module.css'
+
+/**
+ * A <button> version of "components/inline-link",
+ * to support onClick navigation actions such as window.history.back()
+ */
+function InlineLinkButton({
```

Comment:
From designs, we seem need a link-looking element that can call `router.back()`. So I figured it made sense to add a `<button />` styled like an inline link. Styles are quite duplicative of `components/inline-link`, but felt better to copy and paste than to mash together a more complex abstraction.

Reply:
🚧 I know we're still having a convo with design about this in Slack, but just to reiterate here for posterity:I feel very strongly that we do not place an element that looks identical to an inline link, inline with text, but code it as a button. This is a very large accessibility concern; larger than a link a that looks like a button. Our safest bet in this case is to use plain text and iterate on the page + a "go back" element that behaves like the browser back button.

Reply:
@ashleemboyer I've updated for `go back` to be plain text, for now at least, and have removed the `inline-link` component!

Definitely looking forward to getting more design input on this front (see Slack thread), but I'm very on board with having unlinked text here; the error page design still feels _way_ better than before since it has nice card-links.

Reply:
Agreed! It looks much better. 😊